packages/sdk/src/types.ts

```diff
@@ -1078,6 +1078,19 @@ export enum DelegationTargetType {
     Baker = 'Baker',
 }
 
+/**
+ * enum to help in mapping the tag within DelegationTargetType in ConfigureDelegation
+ */
+export enum DelegationTargetTypeNumeric {
+    Passive = 0,
+    Baker = 1,
+}
+
+export const mapTagToType: DelegationTargetType[] = [
+    DelegationTargetType.PassiveDelegation, // index 0
+    DelegationTargetType.Baker, // index 1
+];
```

Comment:
I would move this to `serializationHelpers.ts` instead to avoid exposing it publicly to consumers of the SDK